### DISCUSSION TITLE
Matrix Synapse + Element LXC

### DISCUSSION THREAD

**Discussion by ygibson86** [tteck/Proxmox#687] [Ideas]
Thanks a lot for your work.  Makes life very easy!  Is it possible to make a script for Matrix Synapse + Element LXC

**Comment by ianalexis** [comment]
Try YunoHost. The packages are available.

**Reply by jsapede** [reply]
yunohost is powerful but pain in the ass to manage subdomain, especially with cloudflared tunnels, as its all  (firewall / nginx / ...) integrated in the distribution / LXC
would be cool to have synapse + postgres standalaone on link to existing postgres server

**Reply by ianalexis** [reply]
Idk if its that hard but i support the idea to make a dedicated Synapse + PG LXC.
Maybe some kind of admin panel to control who can log in to make a private server but dont know any that do that.

**Reply by MatKlein** [reply]
I'm also currently looking for a simple way to get Matrix and Element running under Proxmox. 
I have therefore also voted up.

**Reply by ianalexis** [reply]
Just a safety comment, its not recommended to use a Element client in the same server/domain that synapse.
https://security.snyk.io/vuln/SNYK-PYTHON-MATRIXSYNAPSE-1019356
So just a Synapse + PG + Admin LXC. Maybe another domain isolated for the client (or just use something else).
